vizro-core/docs/pages/development/contributing.md

```diff
@@ -47,13 +47,19 @@ If no example is specified (`hatch run example`) then the [default example](http
 
 ## Testing
 
-Tests are handled using the [`pytest`](https://docs.pytest.org/) framework, and test environments are managed by Hatch. To run all tests, run
+Tests are handled using the [`pytest`](https://docs.pytest.org/) and [`jest`](https://jestjs.io/) frameworks, and test environments are managed by Hatch. To run all tests, run
```

Comment:
```suggestion
Tests are handled using the [`pytest`](https://docs.pytest.org/) and [`jest`](https://jestjs.io/) frameworks, and test environments are managed by Hatch. To run all python tests, run
```

Reply:
Isn't that `hatch run test` runs all tests (python and js)?

I implemented like that in the hatch.toml file:
```
test = [
  "test-unit",
  "test-integration",
  "test-js"
]
```

Reply:
Since I supported passing arguments to the jest through the command `hatch run test-js -all`, I removed calling `test-js` with the command `hatch run test` because `jest` has different arguments and flags than `pytest`.

Does that make sense for you?

Reply:
You are right... Hm, ok let's keep it then